spec/features/merchant_items/show_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'Merchant Items Show Page' do
+  before :each do
+    @merchant1 = create(:merchant)
+    @merchant2 = create(:merchant)
+
+    @item1 = create(:item, enabled: true, merchant: @merchant1, unit_price: 59_400)
+    @item2 = create(:item, enabled: true, merchant: @merchant1, unit_price: 95_658)
+  end
+
+  describe "a merchant's item's show page" do
+    it "can link to the merchant's item's show page" do
+      visit merchant_items_path(@merchant1)
+
+      click_link("#{@item1.name}")
+
+      expect(current_path).to eq(merchant_item_path(@merchant1, @item1))
+    end
+
+    it "show the item's name, description, and price" do
+      visit merchant_item_path(@merchant1, @item1)
+
+      expect(page).to have_content(@item1.name)
+      expect(page).to have_content(@item1.description)
+      expect(page).to have_content('$594.00')
+
+      expect(page).to_not have_content(@item2.name)
+      expect(page).to_not have_content(@item2.description)
+      expect(page).to_not have_content('$956.58')
+    end
```

Comment:
This test is good!  You could consider making the test more dynamic by interpolating the item price instead of hard coding it.

Reply:
I added the string interpolation for both prices. I did leave the '594.00' because it could "pass" but not show both trailing zeroes.